features-json/font-metrics-overrides.json

```diff
@@ -0,0 +1,419 @@
+{
+  "title": "@font-face metrics overrides",
+  "description": "Default font metrics overriding: the ascent-override, descent-override and line-gap-override descriptors",
+  "spec": "https://drafts.csswg.org/css-fonts-4/#font-metrics-override-desc",
+  "status": "unoff",
+  "links": [
+    {
+      "url": "https://docs.google.com/document/d/1PW-5ML5hOZw7GczOargelPo6_8Zkuk2DXtgfOtJ59Eo/edit",
+      "title": "New @font-face descriptors for overriding font metrics"
+    },
+    {
+      "url": "https://www.chromestatus.com/feature/5651198621253632",
```

Comment:
This link will automatically be added thanks to the "chrome_id" value, so it can be removed here.